ebean-core/src/main/java/io/ebeaninternal/server/query/DefaultDbSqlContext.java

```diff
@@ -164,6 +164,14 @@ public void addJoin(String type, String table, TableJoinColumn[] cols, String a1
     if (addAsOfOnClause) {
       sb.append(" and ").append(historySupport.getAsOfPredicate(a2));
     }
+    
+    if (extraWhere != null && !extraWhere.isEmpty()) {
+      sb.append(" and ");
+      // we will also need a many-table alias here
+      sb.append(extraWhere.replace(tableAliasPlaceHolder, a2).replace("${mta}", a1));
```

Comment:
@rbygrave Why is `tableAliasPlaceHolder` specified in the databasePlatform?
Should we specify `${mta}` also in the db-platform or should we refactor databasePlatform and remove tableAliasPlaceHolder. As long as I see, it is always `${ta}` and never changed...

Reply:
That is probably the case.  I need to review that.

Reply:
> Why is tableAliasPlaceHolder specified in the databasePlatform?

It is not actually.  Just defined locally as a static final.  We can do the same with `${mta}`